src/main/scala/io/iohk/ethereum/network/AuthHandshaker.scala

```diff
@@ -15,12 +15,12 @@ import org.spongycastle.math.ec.ECPoint
 import scorex.core.network
 import scorex.core.network._
 
-class Secrets(
-    val aes: Array[Byte],
-    val mac: Array[Byte],
-    val token: Array[Byte],
-    val egressMac: KeccakDigest,
-    val ingressMac: KeccakDigest)
+case class Secrets(
```

Comment:
Yes, I made it regular class on purpose. Beside the Arrays, KeccakDigest is mutable (as I wrote in my previous PR description).